### DISCUSSION TITLE
Ban

### DISCUSSION THREAD

**Discussion by luisamanett** [Sehelitar/Kick.bot#72] [Q&A]
Hello i need an someone tell me how to set it so  if a user writes a word already included in the list of a command, he is banned for 1 minute and then if he does it again he is permanently banned? thanks

**Comment by mocchimon** [comment]
![image](https://github.com/user-attachments/assets/3af52678-c1d6-47a9-8ca6-328257c7468b)
Here's an example of what I made. In my example I just straight out ban people so with some adjustments you can make a counter to keep track of time outs
